### ISSUE TITLE
Evaluate v9 key bindings. 

### ISSUE THREAD

**Issue by typhonrt** [typhonjs-fvtt/typhonjs-quest-log#147]
Version 9 of Foundry has a registration system for key bindings. This issue tracks the possible integrations that will allow quicker navigation of the quest log. 

One area that I'll be looking into bindings for is the main quest log UI. IE bindings to move quests between categories and / or deleting quests. 

I welcome any other ideas / suggestions on UX / user experience patterns that can benefit from key bindings.

**Comment by brunocalado** [comment]
The Party Overview module have a simple example. You can link a shortcut:
- Show, hide and toogle

For Party Overview I set letter P with toogle. 

For Quest Log I would bind letter Q with toogle too.

**Comment by typhonrt** [comment]
Indeed... show quest log / quest tracker as two bindings makes sense. 

I'm also curious about workflows inside of the quest log. I think in general some of the most relevant key bindings like opening the log / tracker will make the first release of TQL.
